src/pspm_prepdata.m

```diff
@@ -38,6 +38,12 @@
 newsr = 0;
 outdata = data;
 
+% check input for NaN values
+% -------------------------------------------------------------------------
+if any(isnan(data))
+    warning('ID:invalid_input', 'Data contains NaN values.');
+end
+

```

Comment:
Looks good. Could you also say something along the lines of prepdata can't handle nans, set the status to -1 and return immediately?

Reply:
is that what you mean ? (see the new commit)

Reply:
Exactly.

Reply:
I will approve after you remove the `.lyx~` file.

Reply:
Yeah sorry forgot it --'